src/main/java/com/autotune/analyzer/services/CreateExperiment.java

```diff
@@ -171,6 +171,14 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
     protected void doDelete(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
         Map<String, KruizeObject> mKruizeExperimentMap = new ConcurrentHashMap<String, KruizeObject>();
         String inputData = "";
+        String rm = request.getParameter(AnalyzerConstants.ServiceConstants.RM);
+        boolean rmTable = false;
```

Comment:
As the `if` block below is used to just match the value and set the `rmTable`, we could simplify it with 

```
// Check if rm is not null and set to true
boolean rmTable = (null != rm && AnalyzerConstants.BooleanString.TRUE.equalsIgnoreCase(rm.trim()));
```